### PR TITLE
Doc: NTFS simplifications on links with datasets and contributors

### PR DESCRIPTION
**Pull request by prhod**

### CODE REVIEW

**Review by TeXitoi — APPROVED** [review]

File: documentation/ntfs/ntfs_changelog_fr.md [hunk 1]
```diff
@@ -50,3 +50,5 @@
     * feed_infos.txt est optionnel en 0.6
 * Version 0.7.0 du 03/01/2017
     * feed_infos.txt est obligatoire
+    * le lien entre "stop" et "contributor" est supprimé (liaison possible via "vehicle_journey")
+    * le lien entre "vehicle_journey" et "contributor" est supprimée, et se fait via "dataset"
```
Comment (pbougue) [inline documentation/ntfs/ntfs_changelog_fr.md]: supprim`é`

**Review by pbougue — APPROVED** [review]
So we had a redundancy before?
Anyway, looks good.

**Comment by prhod** [comment]
@pbougue more than redundancy, there was two ways of specifying a relation between a stop and a contributor. Now there is only one :)

**Comment by prhod** [comment]
As discussed, I made the removal of fax and the work on postal address of companies and networks

**Review by TeXitoi — APPROVED** [review]

**Pull request merged**